src/main/java/group1/mips_simulator/components/memParts/CacheBlock.java

```diff
@@ -0,0 +1,90 @@
+package group1.mips_simulator.components.memParts;
+
+import group1.mips_simulator.components.Config;
+import group1.mips_simulator.components.Word;
+
+public class CacheBlock extends Memory{
```

Comment:
I'm uncomfortable with the idea that a Cache Block extends Memory. Do you have any tests for this CacheBlock itself?

Reply:
Upon further reflection, I guess this is ok. It's just not how I would have done it. Which is fine! I just need to learn this method a little better.